Makefile

```
@@ -1,8 +1,14 @@
-.PHONY: test
+.PHONY: test compile_test coverage
 
 SUPPORT_LIBRARY = support_libraries
 JAVA_ROOT = build/java
 
-test:
+compile_test:
```

Looking on what it does it is just an alias for compiling, and not really have anything to do with tests.
```suggestion
compile
```